### ISSUE TITLE
Indicate cloud provider on bar chart

### ISSUE THREAD

**Issue by ericksod** [cloud-carbon-footprint/cloud-carbon-footprint#195]
#### Overview
From feedback from our partners, we learned that it would improve their ability to understand the data and take action if they could see which cloud provider each of the data belonged to. This card will add the cloud provider as a label to each of the bar charts.

## Acceptance Criteria
- [x] When I view the service bar chart, I see the cloud provider indicated below the service label (AWS, Azure, GCP)
- [x] When I view the account bar chart, I see the cloud provider indicated below the account label
- [x] When I view the regions bar chart, I see the cloud provider that uses that region indicated below the region label, separated by commas
- [x] When I view the cloud provider, the text is distinguishable in color/size/parenthesis from the region/account/service text
- [x] When I view the label next to the bars, it is vertically centered with the bar or top aligned with the bar
- [ ] ~~When I have only one cloud provider selected, the cloud provider does not show on the bar charts (let's discuss this one!)~~

## Tech Tasks
- [x] Run Changesets, bump any packages that have been updated/changed

**Comment by ccasher** [comment]
@antoinette-lovelace we started on the task but moved it back into the to-do column. we were referencing this:
https://apexcharts.com/docs/multiline-text-and-line-breaks-in-axes-labels/

Our goal was to transform the data so that instead of Account, Region or Service being passed through as a string for the X-value, it would instead be passed through as an array like: ['Account', 'Cloud Provider'],  and that should separate it out to multi line.

We pushed up some changes to the branch: cloud-provider-bar-chart if you would like to reference that.

Thanks!
